src/components/Tag/Tag.vue

```diff
@@ -40,6 +40,10 @@ export default defineComponent({
     color: {
       type: String,
       default: ''
+    },
+    miter:{
+      type: Boolean,
+      default: true
```

Comment:
This is an anti-pattern but necessary for backwards compatibility. At some point we should change the default to false and go add the miter prop to the existing components.